### PR TITLE
Add reproducible build for armhf and arm64 related versions.

### PR DESCRIPTION
**Pull request by liushilongbuaa**
#### Why I did it
Azp for sonic-slave on arm arch is breaking for missing these versions.
Version auto update need more time than upload version manually. And version auto update is still under testing.
#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106

### CODE REVIEW

File: sonic-slave-buster/Dockerfile.j2 [hunk 1]
```diff
@@ -491,6 +491,7 @@ RUN pip3 install git+https://github.com/aristanetworks/swi-tools.git@bead66bf261
 {% if CONFIGURED_ARCH != "amd64" -%}
 # Install node.js for azure pipeline
 RUN curl -sL https://deb.nodesource.com/setup_14.x | bash -
+RUN curl --silent https://deb.nodesource.com/gpgkey/nodesource.gpg.key | sudo apt-key add -
```
Comment (qiluo-msft) [inline sonic-slave-buster/Dockerfile.j2:494]: Why this new line is required now? If it is a generic issue, we need to fix master first then cherry-pick.

File: sonic-slave-stretch/Dockerfile.j2 [hunk 2]
```diff
@@ -439,6 +439,7 @@ RUN pip2 install git+https://github.com/aristanetworks/swi-tools.git@d51761ec0bb
 {% if CONFIGURED_ARCH != "amd64" -%}
 # Install node.js for azure pipeline
 RUN curl -sL https://deb.nodesource.com/setup_10.x | bash -
+RUN curl --silent https://deb.nodesource.com/gpgkey/nodesource.gpg.key | sudo apt-key add -
```
Comment (qiluo-msft) [inline sonic-slave-stretch/Dockerfile.j2:442]: The same

File: sonic-slave-buster/Dockerfile.j2 (hunk 1 above)
Comment (liushilongbuaa) [reply sonic-slave-buster/Dockerfile.j2:494]: This is observed in the following azp job. I will send PR to master first.
https://dev.azure.com/mssonic/build/_build/results?buildId=24827&view=logs&j=594b531d-901a-5eaa-0051-20781037afc4&t=2d2c06c2-f02c-5153-72fa-81e97e4a6a65

Comment (liushilongbuaa) [reply sonic-slave-buster/Dockerfile.j2:494]: but this issue doesn't occur in our official build for sonic-slave-buster. Maybe it is transient? I will take care of it.

**Review by qiluo-msft — APPROVED** [review]

**Pull request merged**